mypy/types.py

```diff
@@ -2050,7 +2050,7 @@ def callable_type(fdef: mypy.nodes.FuncItem, fallback: Instance,
 
 
 def get_typ_args(tp: Type) -> List[Type]:
-    """Get all type arguments from a parameterizable Type."""
```

Comment:
I found this quite interesting, as I always have written parameterizable. Apparently both are acceptable and used in literature, but the more accpted spelling is parametrizable.

Reply:
I was not sure about this change, being a non-native speaker. I should have looked it up, because changing acceptable spellings is just noise. Thanks for this comment! 😸